src/core.js

```diff
@@ -0,0 +1,38 @@
+// TODO: non-working code. Implement after accepting an API.
+
+var modules = {};
+
+function registerModule(command, moduleObject) { 
+    modules[command] = moduleObject;
+}
+
+function unregisterModule(command) {
+    if (modules[command].enabled) {
+        modules[command].onDisable();
+    }
+    delete modules[command];
```

Comment:
Using `delete` looks odd to me. Consider using ES6 `Map` which allows to delete entries.

Reply:
Done.